ha/pcswrap/pcswrap/internal/connector.py

```diff
@@ -260,10 +260,19 @@ def ensure_shutdown_possible(self, node_name: str) -> None:
             raise RuntimeError(
                 f'No stonith resource is found for node {node_name}.')
 
-    def get_eligible_resource_count(self) -> int:
+    def _get_resources_configured_tag(self) -> Any:
         xml_str = self.executor.get_full_status_xml()
         xml = self._parse_xml(xml_str)
-        tag = xml.find('./summary/resources_configured')
+        return xml.find('./summary/resources_configured')
+
+    def get_stopped_resource_count(self) -> int:
+        tag = self._get_resources_configured_tag()
+        disabled = int(tag.attrib['disabled'])
+        blocked = int(tag.attrib['blocked'])
+        return disabled + blocked
```

Comment:
What are the other statuses for resources? A comment mentioning those statuses here will be helpful.
Should 'failed' status be accounted here?

Reply:
@ajaysrivas Here is the "summary" tag that is analyzed here:
```xml
<summary>
        <stack type="corosync" />
        <current_dc present="true" version="1.1.23-1.el7-9acf116022" name="srvnode-1" id="1" with_quorum="true" />
        <last_update time="Wed Dec 23 06:16:25 2020" />
        <last_change time="Tue Dec 22 12:25:25 2020" user="root" client="cibadmin" origin="srvnode-1" />
        <nodes_configured number="2" expected_votes="unknown" />
        <resources_configured number="71" disabled="0" blocked="0" />
        <cluster_options stonith-enabled="true" symmetric-cluster="true" no-quorum-policy="ignore" maintenance-mode="false" />
    </summary>
```

There are no failed counter there. `number` (I guess - total), `disabled` and `blocked` only. Do you think it is worth adding this XML snippet as is to the comment?

Reply:
A one line text explaining the states would be good.
I assume that there are no optional states in resource_configured tag.

Reply:
@ajaysrivas The comment is addressed.